test/blocks/global-navigation/global-navigation.test.js

```diff
@@ -229,12 +229,27 @@ describe('global navigation', () => {
     });
 
     it('doesn\'t exist on mobile', async () => {
-      const promoMeta = toFragment`<meta name="gnav-promo-source" content="http://localhost:2000/fragments/correct-promo-fragment">`;
+      const promoMeta = toFragment`<meta name="gnav-promo-source" content="http://localhost:2000/fragments/hide-promobar-on-mobile">`;
       document.head.append(promoMeta);
       const nav = await createFullGlobalNavigation({ viewport: 'mobile', hasPromo: true });
-      expect(nav.block.classList.contains('has-promo')).to.be.false;
+      expect(nav.block.classList.contains('has-promo')).to.be.true;
       const asideElem = nav.block.querySelector('.aside.promobar');
-      expect(asideElem).to.not.exist;
+      expect(asideElem).to.be.exist;
+      const prmobarNotOnMobile = asideElem.querySelector('.mobile-up').classList.contains('hide-block');
+      expect(prmobarNotOnMobile).to.be.true;
+      promoMeta.remove();
+    });
+
+    it('Exist on mobile', async () => {
```

Comment:
I think we could have more descriptive test names that tell the reader when the promobar should or should not exist on mobile, i.e. when we have content for mobile